src/scss/components/_logosSection.scss

```diff
@@ -17,10 +17,10 @@
     height: 148.31px;
     position: absolute;
     top: 468.64px;
-    left: 349px;
+    right: -85px;
   }
 
-  @include breakpoint(md) {
+  @include breakpoint(lg) {
```

Comment:
There is an issue between 1024px to 1439px,
you can fix it by changing the breakpoint "lg" to "md"